cmd/rookd/mon.go

```diff
@@ -37,53 +34,35 @@ var monCmd = &cobra.Command{
 }
 
 var (
-	cluster mon.ClusterInfo
 	monName string
 	monPort int
 )
 
 func init() {
 	monCmd.Flags().StringVar(&monName, "name", "", "name of the monitor")
-	monCmd.Flags().StringVar(&cluster.FSID, "fsid", "", "keyring for secure monitors")
-	monCmd.Flags().StringVar(&cluster.MonitorSecret, "mon-secret", "", "keyring for secure monitors")
-	monCmd.Flags().StringVar(&cluster.AdminSecret, "admin-secret", "", "keyring for secure monitors")
 	monCmd.Flags().IntVar(&monPort, "port", 0, "port of the monitor")
 
+	flags.SetFlagsFromEnv(monCmd.Flags(), "ROOKD")
```

Comment:
in my testing, the env vars for persistent flags are initialized with the rootCmd and we don't need to initialize them here as well